cli/commands/disable/index.ts

```diff
@@ -19,12 +19,12 @@ export default function provideDisable(
   return async function disable() {
     const agentExists = await agentRegistry.agentExists(agentId)
     if (!agentExists) {
-      throw new Error(`agent id ${agentId} does not exist`)
+      throw new Error(`bot id ${agentId} does not exist`)
```

Comment:
lets also rename any variables containing "agent" to "bot" i.e. `botId`

Reply:
Renamed all variables to use bot instead of agent